packages/reports/addon/components/filter-values/dimension-select.ts

```diff
@@ -80,7 +83,17 @@ export default class DimensionSelectComponent extends Component<DimensionSelectC
 
   @action
   setValues(dimension: NaviDimensionModel[]) {
-    const values = dimension.map(({ value }) => value) as (string | number)[];
+    const values = [] as (string | number)[];
```

Comment:
Does this method need to know about `manualQuery`? Can it just get a list of values like before?